agents/pipeline_monitor.py

```
@@ -373,6 +375,90 @@ def _analyze_task_status(self) -> Dict[str, Any]:
+        """Ingest recent processing_status.jsonl entries and update OCR metrics.
+
+        Returns an OCR summary dict and also updates internal metrics and alerts.
+        """
+        p = Path(status_path)
+        if not p.exists():
+            return {"error": "status file not found"}
+
+        # read last N lines
+        lines = []
+        with p.open('r', encoding='utf-8') as fh:
+            for l in fh:
```

Ruff (0.14.10)

Ambiguous variable name: `l`

(E741)